packages/graphback-codegen-resolvers/src/ResolverGeneratorPlugin.ts

```diff
@@ -92,7 +92,7 @@ export class ResolverGeneratorPlugin extends GraphbackPlugin {
         const generatedResolvers = createResolverTemplate(resolvers, pluginConfig);
 
         return {
-            fileName: `resolvers.${pluginConfig.format}`,
+            fileName: `${pluginConfig.resolversFileName}.${pluginConfig.format}`,
```

Comment:
🙉

Reply:
Should this be reverted?

Reply:
No.. Just the other guy who write code needs to slow down and start reading their own code. So many silly mistakes there :D